src/sidebar/views/overview.ts

```diff
@@ -16,91 +16,87 @@ export class OverviewItem {
-    private items: {[id: string]: OverviewItem} = {
-        'loading': new OverviewItem('Loading overview, please wait...'),
-
-        'notfound': new OverviewItem('CodeChecker run not found.'),
-        'notfound2': new OverviewItem('Run CodeChecker, reload metadata,'),
-        'notfound3': new OverviewItem('or set the output folder to get started'),
-
-        'files': new OverviewItem(() => `Total files analyzed: ${ExtensionApi.metadata.sourceFiles.size}`),
-        'lastRun': new OverviewItem(() => `Last analysis run: ${
-            new Date(ExtensionApi.metadata.metadata!.timestamps.begin /* seconds */ * 1000).toLocaleString()
-        }`),
-        'buildLength': new OverviewItem(() => {
-            // Time in seconds
-            const beginTime = ExtensionApi.metadata.metadata!.timestamps.begin;
-            const endTime = ExtensionApi.metadata.metadata!.timestamps.end;
-            const interval = endTime - beginTime;
-
-            const hours = Math.floor(interval/3600) % 60;
-            const minutes = Math.floor(interval/60) % 60;
-            const seconds = Math.floor(interval) % 60;
-            const ms = Math.floor(interval * 1000) % 1000;
-
-            if (hours > 0 || minutes > 0) {
-                // H:MM:SS s / M:SS s
-                const formattedMinutes = hours > 0 ? (`${hours}:${minutes.toString().padStart(2, '0')}`) : minutes;
-                return `Analysis duration: ${formattedMinutes}:${seconds.toString().padStart(2, '0')} s`;
-            } else {
-                // S.fff s
-                return `Analysis duration: ${(seconds + ms / 1000).toFixed(3)} s`;
-            }
-        }),
-        'analyzers': new OverviewItem(() => `Used analyzers: ${
-            Object.keys(ExtensionApi.metadata.metadata!.analyzers).join(', ')
-        }`),
-
-        'separator': new OverviewItem('——'),
-
-        'reloadMetadata': new OverviewItem('Reload CodeChecker metadata', {
-            title: 'reloadMetadata',
-            command: 'codechecker.backend.reloadMetadata',
-        }),
-        'analyzeCurrentFile': new OverviewItem('Re-analyze current file', {
-            title: 'reloadMetadata',
-            command: 'codechecker.executor.analyzeCurrentFile',
-        }),
-        'analyzeProject': new OverviewItem('Re-analyze entire project', {
-            title: 'reloadMetadata',
-            command: 'codechecker.executor.analyzeProject',
-        }),
+export class OverviewView implements TreeDataProvider<OverviewItem> {
+    private tree?: TreeView<OverviewItem>;
+
+    private topItems: {[id: string]: OverviewItem[]} = {
+        'loading': [
+            new OverviewItem('Loading overview, please wait...')
+        ],
+        'notFound': [
+            new OverviewItem('CodeChecker run not found.'),
+            new OverviewItem('Run CodeChecker, reload metadata,'),
+            new OverviewItem('or set the output folder to get started')
```

Comment:
```suggestion
            new OverviewItem('Note: CodeChecker is not run yet. Please use commands below to get started.')
```

With the `Note` prefix we will indicate that this is just a note text and it is not clickable.

Reply:
The sidebar items don't wrap, so we're limited by line width here.
And since there are no buttons in that group, even when statistics are displayed, I'm not in favour of putting `Note` everywhere.

Maybe in the future the same kind of dialog could be shown for CodeChecker analyze, or perhaps some other way of walking the user through setting it up.